pkg/xds/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018 Istio Authors
```

Comment:
Most of the code goes under pilot/pkg/.../v2 - any reason for RDS to be at top level ?

Reply:
@nmittler 
@ZackButcher